src/datagrid/DataGridValues.tsx

```diff
@@ -428,16 +447,50 @@ export const pageFilterFunction = (
 
         // Purposefully added dealy here to see loading spinner
         setTimeout(function() {
-            console.log("rows", rows);
             resolve(result);
         }, 2000);
     });
 };
 
-// Data for Hide/show columns
-export const hideableColumns = [
-    {columnName: "User ID"},
-    {columnName: "Name"},
-    {columnName: "Creation Date", isVisible: false},
-    {columnName: "Favorite color", isVisible: false},
-];
+/**
+ * Data for Batch Actions
+ */
+// Grid Action component
+type GridActionsState = {
+    selectedRows: any[];
+    showEdit: boolean;
+};
+
+export class GridActions extends React.PureComponent<any, GridActionsState> {
+    state = {
+        selectedRows: [],
+        showEdit: false,
+    };
+
+    updateActions(rows: any) {
+        this.setState({
+            selectedRows: rows,
+            showEdit: rows.length === 1 ? true : false,
+        });
+    }
+
+    render() {
+        const {selectedRows, showEdit} = this.state;
+        return (
+            <div>
+                <Button key="new">NEW</Button>
+                <Button key="edit" show={showEdit}>
+                    EDIT
+                </Button>
+                <Button
+                    key="delete"
+                    onClick={() => {
+                        alert("Deleted" + selectedRows.length);
```

Comment:
Why we are adding this Alert etc ? 
The code looks to be not story book code, why do we have so much hard coding here ?

NEW, 
"Deleted"
DELETE

Reply:
@maheshkulkarni4 This code is for story only. It is for batch action story. Hence hardcoded.